doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2464 @@
+
+If you start with this array:
+
+::
+
+  a = np.arange(6)
+  print(a)
+
+**Output:**
+
+::
+
+  [0 1 2 3 4 5]
+
+You can use ``reshape()`` to reshape your array. For example, you can reshape this array to an array with three rows and two columns:
+
+**Input:**
+
+::
+
+  b = a.reshape(3,2)
+  print(b)
+
+**Output:**
+
+::
+
+  [[0 1]
+   [2 3]
+   [4 5]]
+
+With ``np.reshape``, you can specify a few optional parameters:
+
+**Input:**
+
+::
+
+  numpy.reshape(a, newshape, order)
+
+``a`` is the array to be reshaped.
+
+``newshape`` is the new shape you want. You can specify an integer or a tuple of integers. If you specify an integer, the result will be an array of that length. The shape should be compatible with the original shape.
+
+``order:`` ``C`` means to read/write the elements using C-like index order,  ``F`` means to read/write the elements using Fortran-like index order, ``A`` means to read/write the elements in Fortran-like index order if a is Fortran contiguous in memory, C-like order otherwise. (This is an optional parameter and doesn't need to be specified.)
+
+If you want to learn more about C and Fortran order, you can `read more about the internal organization of NumPy arrays here <https://numpy.org/devdocs/reference/internals.html>`_. Essentially, C and Fortran orders have to do with how indices correspond to the order the array is stored in memory. In Fortran, when moving through the elements of a two dimensional array as it is stored in memory, the **first** index is the most rapidly varying index. As the first index moves to the next row as it changes, the matrix is stored one column at a time. This is why Fortran is thought of as a **Column-major language**. In C on the other hand, the **last** index changes the most rapidly. The matrix is stored by rows, making it a **Row-major language**. What you do for C or Fortran depends on whether it's more important to preserve the indexing convention or not reorder the data.
+
+`Learn more about shape manipulation here <https://numpy.org/devdocs/user/quickstart.html#shape-manipulation>`_.
+
+
+How to convert a 1D array into a 2D array (how to add a new axis to an array)
+-----------------------------------------------------------------------------
+
+::
+
+  np.newaxis
+  np.expand_dims
+
+-----
+
+You can use ``np.newaxis`` and ``np.expand_dims`` to increase the dimensions of your existing array.
+
+Using ``np.newaxis`` will increase the dimensions of your array by one dimension when used once. This means that a **1D** array will become a **2D** array, a **2D** array will become a **3D** array, and so on. 
+
+For example, if you start with this array:
+
+::
+
+  a = np.array([1, 2, 3, 4, 5, 6])
+  a.shape
+
+**Output:**
+
+::
+
+  (6,)
+
+You can use ``np.newaxis`` to add a new axis:
+
+**Input:**
+
+::
+
+  a2 = a[np.newaxis, :]
+  a2.shape
+
+**Output:**
+
+::
+
+  (1, 6)
```

Comment:
Is there a reason for using this style instead of the more conventional
```
>>> a2 = a[np.newaxis, :]
>>> a2.shape
(1, 6)
```
I think for the sake of ramping the user up to read the rest of our docs, it would be good to introduce this style almost immediately, perhaps after a quick sentence explaining how to read it

Reply:
Hello! I'd be happy to do that if that's the general consensus! I chose this style because it's a lot easier for beginners to understand at a glance, but if it makes more sense to go that direction, I'd be happy to put that together.

Reply:
Some other reasons to do it the way I suggest:

* CI will check if the output is what you say it is
* We can use the same plugin as scipy to make the prompt disableable in-browser: https://docs.scipy.org/doc/numpy/user/misc.html#examples

Reply:
I would say yes, it'd be better to change to `>>>` and add a paragraph near the beginning to explain that `>>>` is input and everything under it without `>>>` is output.

Reply:
Sounds good!

Reply:
Got it!

Reply:
It's done!